Makefile

```diff
@@ -14,3 +16,27 @@ package:
 .PHONY: docker
 docker:
 	docker build -t jenkins/pct .
+
+tmp:
+	mkdir tmp
+
+tmp/jenkins.war: tmp
+	curl -fsSL http://mirrors.jenkins.io/war/latest/jenkins.war -o tmp/jenkins.war
+
+tmp/javax.activation.jar: tmp
+	curl -fsSL http://central.maven.org/maven2/javax/xml/bind/jaxb-api/2.3.0/jaxb-api-2.3.0.jar -o tmp/jaxb-api.jar
```

Comment:
I don't know the command offhand, but I think maven has a way to download dependencies by group/artifact/version. That way you can reuse whatever maven mirrors are configured locally.

Reply:
With `curl`, you're hard-coding the mirror. With `mvn`, it will use whatever mirror is configured in maven's settings.

Reply:
true, but you spent more time in maven stuff than on downloading the file, as you are in a maven project, it checks some stuff

Reply:
😆 good point!

Reply:
Should be solved now